.env

```diff
@@ -25,4 +25,4 @@ DISABLE_ENTERPRISE_LOGIN=''
 REGISTER_CONVERSION_COOKIE_NAME=null
 ENABLE_PROGRESSIVE_PROFILING=''
 MARKETING_EMAILS_OPT_IN='false'
-COLLECT_YEAR_OF_BIRTH='true'
```

Comment:
Why did we remove the value here?

Reply:
I believe the default is always true, we will disable it through the remote config.

Reply:
Those were for different purposes and an empty string makes sense in those cases, but in our case, we need to keep the field enabled in default behavior and only disable it for edX and which should be done through the remote config.